modules/ROOT/examples/reliability-patterns-http-to-vm.xml

```diff
@@ -0,0 +1,28 @@
+<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" >
+	<http:listener-connection host="0.0.0.0" port="8081" />
+</http:listener-config>
+<vm:config name="VM_Config" doc:name="VM Config" >
+	<vm:queues >
+		<vm:queue queueName="toTransactionalVM" queueType="PERSISTENT"/>
+	</vm:queues>
+</vm:config>
+<flow name="reliable-data-acquisition">
+<http:listener config-ref="HTTP_Listener_config" path="transactionalEndpoint"/>
+<vm:publish config-ref="VM_Config" queueName="toTransactionalVM" sendCorrelationId="ALWAYS"/> <!--1-->
+</flow>
+
+<!-- This is the application logic flow in the reliability pattern.
+	It is a wrapper around a sub-flow, "business-logic-processing". -->
+<flow name="main-flow">
+	<vm:listener doc:name="Listener" config-ref="VM_Config" queueName="toTransactionalVM"
+		transactionalAction="ALWAYS_BEGIN"/> <!--2-->
+	<flow-ref name="business-logic-processing"/>
+</flow>
+
+<!-- This is where the real work of the application happens. -->
```

Comment:
What does "real work" mean in this context?
Are you saying that the app doesn't "really" do anything outside of this section?